app/viewers/embed/viewer/document_viewer.rb

```diff
@@ -27,15 +27,11 @@ def fullscreen?
         true
       end
 
-      def all_documents_location_restricted?
-        document_resource_files.all?(&:location_restricted?)
-      end
-
-      # this indicates if the PDF is downloadable (though it could be stanford only)
-      # Stanford only and location restrictions are handled via a separate authorization flow,
-      # since it is possible for people to do something about the restriction
+      # This indicates if the first PDF is downloadable (though it could be stanford only)
+      # as well as location restricted.  Authorization for other documents will be
+      # checked as the user clicks on items in the content side bar.
       def available?
-        document_resource_files.first&.downloadable?
+        document_resource_files.first&.downloadable? || document_resource_files.first&.location_restricted?
```

Comment:
I'm not sure why we're still checking this on the backend, since it looks like the frontend is also doing this check.

Reply:
I think we keep doing as a patch because I don't think haven't added the JS to display this. this it should be `!document_resource_files.first&.no_download?` but yes I think we should be fixing this in another PR to the file_auth

Reply:
I will leave a comment here to explain why. The code puts up a generic "content not available" block if it thinks the item is not available, hence this change to ensure an object that is not downloadable but is location restricted would not immediately display the content not available block.  @dnoneill I can try your suggestion if it covers all the cases, but that's the main motivation (to not show the content not available block)

Reply:
I will use that logic then, thank you!